md/migrate-a-form-from-6-x.md

```diff
@@ -18,3 +18,8 @@ You can do so with a process that uses a connector to retrieve external data and
 6. In the Studio, at pool level and for each human task, go to the **_Details_** panel, **_Execution_** tab, **_Form_** section, and change the option from **Legacy (6.x)** to **_UI Designer_**. You can also generate them in another development environment. In this case, choose **_External URL_** option.
 
 When you export a diagram, forms created in the UI Designer are embedded by default in the .bos and .bar files, to share between Bonita Studios (.bos) or install in a Bonita Portal (.bar), so you don't need to manually export them and import them one by one.
+
+7. Confirmation page. After a 6.x form was submitted, the Portal handled a Confirmation page that browsed for the next available task in the current case for the logged user. In case there was no such a subsequent task for the user, she needed to go back to the Task List with a click, which happened most of the time. 
+Bonitasoft believes in the huge user experience potential of Living Applications to better empower users who need to follow the management of business objects along a process instance. This is why such use cases (several sequential tasks performed on the same process instance by the same actor) are now implemented with Living Applications.
+Using the Task List is now best fit to manage tasks from several processes, sorted by to due date and priority.
```

Comment:
to manag**ing** ?

Reply:
"Bonitasoft believes in the huge user experience potential": In my opinion this is way too much a marketing message.
"follow the management of business objects along a process instance" is not easy to understand for me.

I think we should clearly highlight that if the process definition include consecutive tasks that belongs to a single user it will actually need to get through the list of task to find the next task.